packages/http/src/validator/validators/utils.ts

```diff
@@ -15,17 +15,15 @@ export const convertAjvErrors = (errors: NonEmptyArray<ErrorObject>, severity: D
     errors,
     map<ErrorObject, IPrismDiagnostic>(error => {
       const allowedParameters = 'allowedValues' in error.params ? `: ${error.params.allowedValues.join(', ')}` : '';
-      const errorPath = error.dataPath.includes('.')
-        ? error.dataPath.split('.').slice(1)
```

Comment:
@XVincentX I'm not quite sure where did the initial `slice(1)` come from. And what requirement it was fulfilling.
Switching to a `split('.')` call seems to still not break any tests while making the code a bit easier to reason with.